CONTRIBUTING.md

```diff
@@ -190,18 +192,54 @@ The "pre-commit" hook runs the [verify-commit](https://github.com/aws-amplify/am
 
 #### "pre-push" hook:
 
-The "pre-push" hook will build test files and run the  `split-e2e-tests` script to ensure the correct configuration file is generated for our CICD workflow.
+The "pre-push" hook will build test files and run the `split-e2e-tests` script to ensure the correct configuration file is generated for our CICD workflow.
 
 ## Tests
 
-Please ensure that your change still passes unit tests, and ideally integration/UI tests. It's OK if you're still working on tests at the time that you submit, but be prepared to be asked about them. Wherever possible, pull requests should contain tests as appropriate. Bugfixes should contain tests that exercise the corrected behavior (i.e., the test should fail without the bugfix and pass with it), and new features should be accompanied by tests exercising the feature.
+Please ensure that your change still passes unit tests. It's OK if you're still working on tests at the time that you submit, but be prepared to be asked about them. Pull requests should contain tests as appropriate. Bugfixes should contain tests that exercise the corrected behavior (i.e., the test should fail without the bugfix and pass with it), and new features should be accompanied by tests exercising the feature.
 
 To run the test suite:
 
 ```bash
 yarn test
 ```
 
+Unit tests can be run on a per-project basis:
+
+```bash
+yarn nx test amplify-category-function
```

Comment:
I wish we could `yarn test amplify-category-function` or something like this. I.e. don't `yarn` and `nx` at the same time.